content/docs/studio/troubleshooting.md

```diff
@@ -89,28 +98,28 @@ If you did not intend to work with a DVC repo, you can also specify custom files
 that contain the metrics and hyperparameters that you want to visualize.
 
 Instructions on how to specify the sub-directory or custom files can be found
-[here](/doc/studio/user-guide/views/view-settings#configuring-view-settings).
+[here](/doc/studio/user-guide/projects/project-settings#configuring-project-settings).
 
-## View got created, but does not contain any data
+## Project got created, but does not contain any data
```

Comment:
Also, make sure to check for `#anchor` links to any headers you changed that may need an update, please!

In this example it would be `/doc/studio/troubleshooting#view-got-cre...` -> `.../troubleshooting#project-got-...`

☝🏼 in fact this probably affects the Studio core codebase (in the case of this page, at least) ?

Reply:
Checked. Thanks.